app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -19,5 +19,6 @@
 }
 
 $govuk-assets-path: '/customs-movements/assets/';
+$govuk-assets-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/";
```

Comment:
Is this a genuine change this time?

Reply:
Does this fix the font issue?

Reply:
No. This is wrong. Thank you for spotting this.